tox.ini

```diff
@@ -41,10 +42,14 @@ allowlist_externals =
                       sh
                       sed
                       mkdir
+                      chmod
+
 commands_pre =
          pip-compile --resolver=backtracking --output-file {envdir}/requirements.txt tests/requirements.txt requirements/base.txt requirements/optional.txt requirements/django{env:DJANGO_VER}.txt
          pip-sync {envdir}/requirements.txt
          pip install -e .
+         mkdir -p {toxinidir}/reports/coverage
+         chmod 777 {toxinidir}/reports/coverage
```

Comment:
Were these automatically created before or have they been manually moved?

Reply:
No, only the `{toxinidir}/reports` part is typically created as part of a test run from before. This adds the extra subdirectory `coverage`, and tells coverage to put its coverage database files (primarily an SQLite3 file called `.coverage`) there, while making sure all subprocesses can write to it.